classes/02class/exercises/c02-network08/deniseddi/c02-network08.txt

```diff
@@ -0,0 +1,18 @@
+
+Network ACL (NACL's)
+
+•	What is the difference between inbound x outbound?
+    Both have to do with the direction of traffic. 
+    Inbound: traffic initiate from external.
+    Outbound: traffic initiate from internal.
+    Therefore, inbound rules filter traffic passing (in) to the subnet and outbound rules filter traffic passing (out) from the subnet to the network. 
+
+•	What is going to happen by default? Everything allowed or denied?
+    According to the AWS docs, the default network ACL is configured to allow all traffic to flow in and out of the subnets. So, everything allowed. 
+
+•	How would you block SSH connection coming from the network 172.16.200.200/24?
+    Apparently, the only way to deny sources/IP addresses is to use Network ACL's in the VPC. So, after logging in to AWS console, I would go to my VPC dashboard and then to Network ACLs. I can then open the ACL editor, select the subnet where I would have my instance running. Edit the inbound rules by adding a new rule and explicitly “deny” the IP address 172.16.200.200/24 on port 80 (or any number less than 100, in order to be evaluated before the default). Save it. 
+
+•	How would you block access from your network to an external network 180.200.50.0/24?
+    I would go to my VPC dashboard and then to Network ACLs. Next, open the ACL editor and  select the subnet, this time selecting the Outbound rules -> Edit it by adding a new rule and explicitly “deny” the IP address 180.200.50.0/24 on a port number less than 100. Save it.
+
```

Comment:
The number you're referring as the port number is in fact the Rule number and not the port.

In an ACL rule you need to specify the number so it the rules can be evaluated in the order you want. You are right that the number needs to be below 100, but that's not the port.
One of the options for an ACL rule is the protocol and the  port-range that will be affected by that rule, so those things are also part of the acl rule, but since we want to block any access from that specific subnet, the protocol that we need t specify is `ANY` and the port range is also `ANY`.

Please, adjust your answer.